guides/interacting-with-pages.rst

```diff
@@ -0,0 +1,139 @@
+Interacting with Pages
+======================
+
+Most usages of Mink will involve working with the page opened in your browser.
+The Mink Element API lets you interact with elements of the page.
+
+Interacting with Links and Buttons
+----------------------------------
+
+The ``NodeElement::click`` and ``NodeElement::press`` methods let you action
```

Comment:
What `action links` means? Maybe you wanted to say `act on links`. The `act` term is too wide. We can rewrite whole sense to:
```
The ``NodeElement::click`` and ``NodeElement::press`` methods let you click the links and press the buttons on the page.
```